perses/annihilation/relative.py

```diff
@@ -444,10 +463,22 @@ def _determine_atom_classes(self):
             hybrid_idx = self._new_to_hybrid_map[atom_idx]
             atom_classes['unique_new_atoms'].add(hybrid_idx)
 
-        core_atoms, environment_atoms = self._get_core_atoms()
+        #The core atoms:
```

Comment:
perception of core atoms directly in `TopologyProposal` removes the need for `_get_core_atoms`